sdk/keyvault/keyvault-admin/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
-## 1.0.0-preview.1 (Unreleased)
+## 4.0.0-beta.1 (Unreleased)
 
-- Produced the generated code for the administrative Key Vault tasks. It includes the backup and restore operations for whole Key Vault instances, and the role-based access control (RBAC) operations.
+The @azure/keyvault-admin package provides two clients, `KeyVaultAccessControlClient` and `KeyVaultBackupClient`.
+
+- The `KeyVaultAccessControlClient` allows assigning, deleting and retrieving role assignments, and retrieving role definitions.
+- The `KeyVaultBackupClient` allows generating full backups, selective restores of specific keys and full restores of Key Vault instances.
```

Comment:
nit: you can try condensing by just saying that it does vault-level backup and restore

Reply:
Isn't what you're saying in this case the same as what I'm saying? 🤔

Reply:
> The `KeyVaultBackupClient` allows generating full backups

Full backups of the entire Key Vault instances? If so, I would combine this with the "full restores of Key Vault instances" before talking about specific keys

```suggestion
- The `KeyVaultBackupClient` allows generating full backups and restores of Key Vault instances, and selective restores of specific keys.
```

Is the "specific keys" restricted by the client or any key can be restored? If it is the latter, then I would drop the word "specific"

Also, I dont see `KeyVaultBackupClient` in https://github.com/Azure/azure-sdk-for-js/blob/1e1aa559e12719218e82e06beffeeaa845d914ae/sdk/keyvault/keyvault-admin/review/keyvault-admin.api.md

Reply:
The backup client is here: https://github.com/Azure/azure-sdk-for-js/pull/11009

> The `KeyVaultBackupClient` allows generating full backups and restores of Key Vault instances, and selective restores of specific keys.

Ohh I really like that 😮! Thank you!

I'll drop specific!